cards/hip-injuries-femoral-head-fracture/card.md

```diff
@@ -0,0 +1,49 @@
+---
+
+title: Hip Injuries - Femoral Head Fracture
```

Comment:
Can we choose a format and stick to it for these types?

Other cards that exist (I think ultrasound?) use colons. I prefer the dash as well (colons need to be escaped in yaml).